pkgs/os-specific/linux/musl/default.nix

```diff
@@ -149,7 +149,13 @@ stdenv.mkDerivation rec {
     homepage    = "https://musl.libc.org/";
     changelog   = "https://git.musl-libc.org/cgit/musl/tree/WHATSNEW?h=v${version}";
     license     = licenses.mit;
-    platforms   = platforms.linux;
+    platforms   = [
+      "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux"
+      "armv7l-linux" "i686-linux" "x86_64-linux" "m68k-linux"
+      "microblaze-linux" "microblazeel-linux" "mipsel-linux"
+      "mips64el-linux" "powerpc64-linux" "powerpc64le-linux"
+      "riscv64-linux" "s390x-linux"
```

Comment:
Not sure about `riscv64-linux`...

Reply:
Ah, nice. Maybe `useMusl = !stdenv.targetPlatform.isRiscV` in `pkgs/stdenv/linux/make-bootstrap-tools.nix` could be updated.

Reply:
Done.